tox.ini

```diff
@@ -23,6 +23,7 @@ deps =
     pinned: six==1.10.0
     pinned: werkzeug==0.11.11
     pinned: wrapt==1.10.8
+    pinned: regex==2018.2.21
```

Comment:
Not 100% sure how this works, but was this supposed to be `extra: regex==2018.2.21`?